### ISSUE TITLE
great program

### ISSUE THREAD

**Issue by ghost** [ReStartQ/anicour#1]
seems to be the only program that covers offline anime+manga+light novel tracking. that's really nice. I've spent many hours searching for a program like this one (many days if you consider my other attempts). Would be nice to have MAL and/or Anidb support but I guess I can simply export my list to Anilist and use it. thanks for this.

**Comment by ghost** [comment]
btw, please make it so that you can add and edit entries manually, regardless of anilist or some other site. that would instantly turn this program into a must-have for many people

**Comment by ghost** [comment]
well, when I stop to think about it, makes no sense for it to only work with anilist. if you can only update it with internet access, what's the point? it's the same as exporting an anilist database.

**Comment by ReStartQ** [comment]
@cyberhardt 
Due to popularity, adding support for MAL is definitely part of the plan in the very near future.
It may take a bit longer to add the AniDB service as I haven't read up on their API yet. (far future)

As of right now, I've been working on a new app to replace this old one.
After I release the new version of the app, I will then work on adding MAL support for it.

The new app is scheduled to be released sometime later this year. (few months)

Here is a preview of what the new app may look like:
![image](https://github.com/ReStartQ/courier/assets/61175330/17aaed7e-2c60-4d1e-b75a-58b1a8f9fd88)

**Comment by ghost** [comment]
that's cool, I'm looking forward to it. 

there is a group of people that either don't use any of those sites or plan on not using them anymore, which is why I think that having the option to add and edit entries manually is a nice thing to have. hope you make this a possibility in the future

**Comment by Cikapkamuy** [comment]
If possible in the new app, could there be inclusion for site specific features such as private entries, adult entries, and support for custom lists?

**Comment by ReStartQ** [comment]
@Cikapkamuy 
I can add support for private and adult entries in the remake. To view adult media in search and seasons results, the user would need to specifically set the adult value as true in the settings menu. Private entries will most likely be a check box or something you can click on in the advanced view of the media. 

With custom lists, its a bit harder to approach this because having more lists that you can show the user would take up a lot of space in the sidebar. I'm not sure on what a good solution to this would be at the moment, but maybe I'll find a good way to implement it in the future. This is something that probably won't be in the initial release of the new app, but may be added in a later version.

**Comment by Skorbis** [comment]
Stumbled upon on this great program by luck, I was using Taiga previously for Windows and that old white UI was always burning my retinas.

Was wondering if it would be possible to request that - / + was added besides progress count in the List view? - Would appreciate the quick way to update episode counts that way.
![image](https://github.com/ReStartQ/anicour/assets/34304889/a8635be5-76b7-4718-8dbb-6b9047717f92)

Love the program, it was a blessing finding this gem

**Comment by ReStartQ** [comment]
@Skorbis 
Yes, I'll try to add this. Coincidentally, I've been working and testing a new headless UI table for the list view because the virtualization for the old table isn't performing very well. The plan for the list view is to make it perform much better in the next version. Adding new features is also one of the goals for this new table.

I'll try to add the -/+ and have it be implemented in the next major version.  (v1.5.0)
It may be a bit difficult to implement this new feature depending on how i approach it. 
If I do run into any issues adding this new feature, it may have to be in a slightly later version. (v1.5.1 or v1.5.2)
